lib/active_admin/views/pages/form.rb

```diff
@@ -44,7 +44,10 @@ def default_form_config
           ActiveAdmin::PagePresenter.new do |f|
             f.semantic_errors # show errors on :base by default
             f.inputs
-            f.actions
+            f.actions do
+              f.action :submit, button_html: { class: "button button-primary" }
+              f.action :cancel, button_html: { class: "button button-default" }
+            end
```

Comment:
I'm confused why my customized forms are getting these classes applied even though they're simply calling `f.actions`, and yet you're manually passing them here. Why is that?